static/js/containers/admin/AdminPage.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+import CreateChannelPage from './CreateChannelPage';
+import Card from '../../components/Card';
+
+import type { Match } from 'react-router';
+
+export default class AdminPage extends React.Component {
+  props: {
+    match: Match,
+  }
+
+  render() {
+    const { match } = this.props;
+    return (
+      <div className="admin double-column">
+        <Card>
+          <h1>Admin</h1>
```

Comment:
should we maybe add a `title` prop to the `Card` component? I'm thinking we may want to style that in a consistent way across the app (just occurred to me, not necessary something to do in this PR)